fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialog.kt

```diff
@@ -65,14 +66,16 @@ fun TranslationOptionsDialog(
             )
         }
 
-        item {
-            TextListItem(
-                label = stringResource(id = R.string.translation_option_bottom_sheet_translation_settings),
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(start = 56.dp),
-                onClick = { onTranslationSettingsClicked() },
-            )
+        if (FxNimbus.features.translations.value().globalSettingsEnabled) {
```

Comment:
I am expecting this to break all the Compose Previews. My recommendation here is to add a suitably named parameter for this composable (eg, `showTranslationSettings = true`) where we pass in these Nimbus features.

Reply:
It has a compile-time default and the Compose Previews render for me. WDYT?

<img width="1445" alt="image" src="https://github.com/mozilla-mobile/firefox-android/assets/93344309/adc9dc27-07ec-4e40-bdbe-d9529347780f">

Reply:
ok no problem from me then.